generated-docs/out/backends/catseffect.md

```diff
@@ -2,6 +2,8 @@
 
 The [Cats Effect](https://github.com/typelevel/cats-effect) backend is **asynchronous**. It can be created for any type implementing the `cats.effect.Concurrent` typeclass, such as `cats.effect.IO`. Sending a request is a non-blocking, lazily-evaluated operation and results in a wrapped response. There's a transitive dependency on `cats-effect`. 
 
+## Using async-http-client
```

Comment:
it seems you've modified the generated documentation - in the `generated-docs` directory. The proper docs live in `docs`, and are generated using `mdoc`. You can then get docs checked by the compiler :)

One note when generating documentation - set the version (`set version := "3.1.1"`) beforehand to avoid getting lots of changes in most doc files

Reply:
Oops... 😅